spec/teaspoon/configuration_spec.rb

```diff
@@ -149,6 +152,19 @@
 
   end
 
+  describe ".coverage_ignored_files" do
+
+    it "returns an empty list if nothing was set" do
+      expect(subject.coverage_ignored_files).to eq([])
+    end
+
+    it "returns an array of  if they were comma separated" do
```

Comment:
If this configuration can't be used on the CLI, I don't think we should support the string version.